src/libpmempool/libpmempool.c

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: BSD-3-Clause
-/* Copyright 2016-2018, Intel Corporation */
```

Comment:
ditto, I think we should at least let pmempool parse poolsets with remote replicas and print out an error.